offgridplanner/steps/views.py

```diff
@@ -28,7 +29,7 @@ def project_setup(request, proj_id=None):
         project = get_object_or_404(Project, id=proj_id)
         if project.user != request.user:
             raise PermissionDenied
-        context = {"project_id": project.id, "max_days": max_days}
+        context = {"form": form, "project_id": project.id, "max_days": max_days}
```

Comment:
Should this not be
```suggestion
        context = {"opts_form": form, "project_id": project.id, "max_days": max_days}
```
as you changed it to `opts_form` in `consumer_selection.html`